apps/service-portal/src/environments/environment.prod.ts

```diff
@@ -13,17 +13,5 @@ export default {
       'https://3c45a55273774b91a897b85e0a1243d1@o406638.ingest.sentry.io/5501494',
   },
   documentProviderAdmins: process.env.NX_DOCUMENT_PROVIDER_ADMINS || '',
-  featureFlags: {
-    applications: false,
-    documents: true,
-    settings: true,
-    finance: true,
-    family: true,
-    health: false,
-    education: false,
-    delegation: false,
-    assets: false,
-    drivingLicense: false,
-    documentProvider: false,
-  },
+  configcatSdkKey: 'YcfYCOwBTUeI04mWOWpPdA/KgCHhUk0_k2BdiKMaNh3qA',
```

Comment:
Change to an environment variable.

Reply:
Yeah my thinking was to have it here in clear text because this is not a secret but there might be a benefit having this in an environment variable so we don't need to repeat it through the projects.
I will change it to env variable.

Reply:
Ah no the reason is I can't get this out of an environment variable as `process.env.X` is evaluated in the browser.
So I at least don't know a good way to solve that easily.

Reply:
This hasnt been solved yet as far as I know.

Reply:
Needs to be set at build time via webpack.

Reply:
Yes because configcat sdk key does not change between environments we can build it into the bundle with something like webpack DefinePlugin.

I am not sure it is a good idea though, because we are adding the CI environment as a dependency to build the project with correct variables. I feel weird about it.

@sindrig thoughts?

Reply:
The SDK indeed changes between environments, we've got different keys for dev, staging and prod! It's available to all pods via the `CONFIGCAT_SDK_KEY` environment variable.

Reply:
:facepalm: ofcourse there are different keys between environments.

Okay then the solution is easy but ugly.